index/visualisation.py

```diff
@@ -30,25 +31,45 @@ def get_cohort_specific_color_code(cohort_name: str):
     elif cohort_name.lower() in COLORS_PD:
         return COLORS_PD[cohort_name.lower()]
     else:
-        print(f'No color code found for cohort {cohort_name}')
+        print(f"No color code found for cohort {cohort_name}")
         return None
 
 
-def enrichment_plot(acc_gpt, acc_fuzzy, title, save_plot=False, save_dir="resources/results/plots"):
-    if len(acc_gpt) != len(acc_fuzzy):
-        raise ValueError("acc_gpt and acc_fuzzy should be of the same length!")
-    data = {"Maximum Considered Rank": list(range(1, len(acc_gpt) + 1)), "GPT": acc_gpt,
-            "Fuzzy": acc_fuzzy}
+def enrichment_plot(
+    acc_gpt,
+    acc_mpnet,
+    acc_fuzzy,
+    title,
+    save_plot=False,
+    save_dir="resources/results/plots",
+):
+    if (
```

Comment:
Could be done a bit shorter ;)

if  **not** len(A) == len(B) == len(C) do stuff